### PR TITLE
added css file and one line is changed in the html file

### PR DESCRIPTION
**Pull request by zekitel**
May be you can contribute on the design, may be we can decide the colors etc.

### CODE REVIEW

File: practice_app/frontend/api.js [hunk 1]
```diff
@@ -32,7 +32,37 @@ function getTweets() {
     }, {
         "id": "2",
         "user": "alesta",
-        "content": "ne dedin sen?",
+        "content": "Sense child d wisdom wcuten whole as match asked. Pleasure exertion put add entrance distance drawings. In equally matters showing greatly it as. Want name any wise are able park when. Saw vicinity judgment remember finished men throwing.",
+        "rt_count": 3,
+        "like_count": 99
```
Comment (aliturgutalp) [inline practice_app/frontend/api.js]: Looking good.

File: practice_app/frontend/api.js [hunk 2]
```diff
@@ -32,7 +32,37 @@ function getTweets() {
...
+        "id": "2",
+        "user": "alesta",
+        "content": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaasdas123123 qwe 123 12aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "rt_count": 3,
+        "like_count": 99
+    }, {
+        "id": "2",
+        "user": "alesta",
+        "content": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaasdas123123 qwe 123 12aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "rt_count": 3,
```
Comment (bartuinceQR) [inline practice_app/frontend/api.js]: We might want to try this out with different field values. Who knows, stuff might break.

Comment (zekitel) [reply practice_app/frontend/api.js]: The field which might be a trouble is the content field. Twitter allows the user to use 280 char. I tried the looking  up to 280 char and arranged the weight and height. Ofcourse we need to try with more realistic test cases.

File: practice_app/frontend/frontend.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  
+  background: #0897f7;
+  background-image: -webkit-linear-gradient(top, #0897f7, #2f7aa8);
+  background-image: -moz-linear-gradient(top, #0897f7, #2f7aa8);
+  background-image: -ms-linear-gradient(top, #0897f7, #2f7aa8);
+  background-image: -o-linear-gradient(top, #0897f7, #2f7aa8);
+  background-image: linear-gradient(to bottom, #0897f7, #2f7aa8);
+  -webkit-border-radius: 18;
+  -moz-border-radius: 18;
+
```
Comment (ergun1017) [inline practice_app/frontend/frontend.css]: we may add `cursor: pointer` here; else, looks good to me

**Review by ergunsh — APPROVED** [review]

**Pull request merged**